src/components/Chart/TimeBars.js

```diff
@@ -584,13 +283,25 @@ export const propTypes = {
   numberFormat: PropTypes.string.isRequired,
   filter: PropTypes.string,
   tLabel: PropTypes.func.isRequired,
-  description: PropTypes.string
+  description: PropTypes.string,
+  column: PropTypes.string,
+  columnSort: sortPropType,
+  columnFilter: PropTypes.arrayOf(
+    PropTypes.shape({
+      title: PropTypes.string.isRequired,
+      test: PropTypes.string.isRequired
+    })
+  ),
+  columns: PropTypes.number.isRequired,
+  minInnerWidth: PropTypes.number.isRequired,
+  skipRowPadding: PropTypes.bool
```

Comment:
Todo for me: look into what this does.

Reply:
the `skipRowPadding just puts padding between the columns, but not at the end of the rows (which makes sense for timebars, but is different for lines/slopes)

Reply:
Ah I see, so more of an internal config? IMO we should not expose it and just hard code `skipRowPadding: true` when calling the util.

Reply:
exactly. i just thought, if you got it, flaunt it. 🤣  
but maybe it adds unnecessary noise indeed.